types/src/content/index.ts

```diff
@@ -28,6 +28,7 @@ export class CuratorGroup extends JoyStructDecorated({
 }) {}
 
 export class ContentActor extends JoyEnum({
+  Collaborator: MemberId,
```

Comment:
The variant ordering must match the one in the runtime, so:
`Curator` -> `Member` -> `Collaborator` -> `Lead`

I was also wondering if it wouldn't be better have `Collaborator` as last variant in both the runtime and `@joystream/types` just in case for extra safety and backward-compatibility (the main rule is we shouldn't change ordering of enum varaints if the enum is stored somewhere in the runtime state, although I it may not be the case in this specific example)